### DISCUSSION TITLE
React Native Windows is incredibly slow to compile on a VM

### DISCUSSION THREAD

**Discussion by shirakaba** [microsoft/react-native-windows#13549] [Q&A] [answered]
It takes me about 30 minutes to compile any React Native Windows app at all (e.g. RNTA) on an Apple M3 Max when running through an ARM Windows VM on Parallels. It's making it incredibly hard to make any progress on anything. By comparison, I recall I was able to build an Electron app in a few minutes.

Is there some sort of trick to developing for Windows on a Mac efficiently? Is Parallels as good an option as any? Are there prebuilds available for RNW? What approach is used at Microsoft?

**Comment by tido64** [comment, marked as answer]
Did you try using the prebuilt binaries? For RNTA, when generating the VS solution, add `--use-nuget`:

```sh
yarn install-windows-test-app --use-nuget
```

If you already have an `ExperimentalFeatures.props`, try adding `<UseExperimentalNuget>true</UseExperimentalNuget>`: https://github.com/microsoft/react-native-test-app/blob/e8edaaa2217aa1191b8a17e35ac1f5065fe98568/windows/ExperimentalFeatures.props#L33

**Reply by shirakaba** [reply]
Thanks, I've tried now – unfortunately, results are inconclusive as I hit this issue https://github.com/microsoft/react-native-windows/issues/13559. The build also sadly seemed to take a very long time even with the prebuilt binaries.
